pyproject.toml

```diff
@@ -101,7 +101,7 @@ mysql = ["sqlalchemy", "pymysql"]
 neo4j = ["neo4j"]
 nginx = []
 opensearch = ["opensearch-py"]
-oracle = ["sqlalchemy", "cx_Oracle"]
+oracle = ["sqlalchemy", "oracledb"]
```

Comment:
Same as [here](https://github.com/testcontainers/testcontainers-python/pull/514/files#r1545495696), should only be a dev dependency (so that we don't include a lot of libs when installing TC)